submissions/MosQuitO404/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,290 @@
+html {
+  box-sizing: border-box;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+
+*, *:before, *:after {
+  font-family: 'DM Sans', sans-serif;
+  box-sizing: inherit;
+  text-decoration: none;
+  font-size: 16px;
+  color: #170F49;
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+.popup-header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 50px;
+}
+
+.nav-items {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  background: #FFFFFF;
+  box-shadow: 0px 3px 14px rgba(74, 58, 255, 0.03), 0px -2px 4px rgba(20, 20, 43, 0.02), 0px 12px 24px rgba(20, 20, 43, 0.04);
+  border-radius: 50px;
+}
+
+@media screen and (max-width: 940px) {
+  .nav-items {
+    border-radius: 20px;
+    padding-top: 10px;
+  }
+  #home-button {
+    order: 1;
+  }
+  #about-button {
+    order: 2;
+  }
+  #pricing-button {
+    order: 3;
+  }
+  #blog-button {
+    order: 4;
+  }
+  #integrations-button {
+    order: 5;
+  }
+  #contacts-button {
+    order: 6;
+  }
+  #popup-wrapper {
+    order: 7;
+    flex-basis: 100%;
+  }
```

Comment:
No need to set order for all elements? you can just set `order: 1` to icon item to move him to the last position in list

Reply:
Done!